### DISCUSSION TITLE
Welcome to ServiceFabric.Mocks Discussions!

### DISCUSSION THREAD

**Discussion by loekd** [loekd/ServiceFabric.Mocks#112] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by zhengmu** [comment]
this is Zheng Mu, working in Azure Core Compute. We are the owner team of Anvil, which uses ServiceFabric for Dev work, and use ServiceFabric.Mocks for testing.
